### ISSUE TITLE
Add help item for how to focus table selection

### ISSUE THREAD

**Issue by lukepighetti** [TaKO8Ki/gobang#164]
After you open a database and select a table, it's not clear how to go back and change the table. Pressing `?` to learn how doesn't help

**Comment by err0r500** [comment]
Agree,  I've the same issue (and I'm still trying to figure out how to do that) 😆

**Comment by Narayanbhat166** [comment]
Yes, this would be an area of improvement. It was not clear that which of the available panes were currently focused and how to navigate.

**Comment by quantonganh** [comment]
Since I'm familiar with [lazygit](https://github.com/jesseduffield/lazygit), and I also don't want to move my hand out of j/k row, I use Tab to move between left and right pane: https://github.com/quantonganh/gobang/blob/main/src/config.rs#L133-L134
